srcpkgs/go/template

```diff
@@ -10,7 +10,7 @@ maintainer="Michael Aldridge <maldridge@VoidLinux.eu>"
 homepage="http://golang.org/"
 license="3-clause-BSD"
```

Comment:
BSD-3-Clause

Reply:
What is the issue? The LICENSE is installed with the package.

Reply:
Or is it just the formatting?

Reply:
https://spdx.org/licenses/BSD-3-Clause